site/content/zh/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Envoy Gateway
+---
+
+{{< blocks/cover title="欢迎访问 Envoy Gateway！" image_anchor="top" height="full" >}}
+<a class="btn btn-lg btn-primary me-3 mb-4" href="/v1.0.0">
+  开始使用 <i class="fas fa-arrow-alt-circle-right ms-2"></i>
+</a>
+<a class="btn btn-lg btn-secondary me-3 mb-4" href="/v1.0.0/contributions">
+  参与贡献 <i class="fa fa-heartbeat ms-2 "></i>
+</a>
+<p class="lead mt-5">将 Envoy 代理作为独立或基于 Kubernetes 的 API 网关进行管理</p>
+{{< blocks/link-down color="white" >}}
+{{< /blocks/cover >}}
+
+{{% blocks/lead color="black" %}}
+将 **Envoy 代理**作为**独立**或**基于 Kubernetes 的** API 网关进行管理。
+
+**Gateway API** 用于**动态**提供和配置托管 Envoy 代理。
+{{% /blocks/lead %}}
+
+{{% blocks/section type="row" color="dark" %}}
+
+{{% blocks/feature icon="fa fa-commenting" title="Expressive API" %}}
+Based on Gateway API, with reasonable default settings to simplify the Envoy user experience, without knowing details of Envoy proxy.
```

Comment:
Since we have added the landing page, can we finish the main page translation complete?

Reply:
Done, the landing page all translated.
<img width="1461" alt="image" src="https://github.com/envoyproxy/gateway/assets/1269496/f0a2b4bc-94b3-4788-881f-b85bb905b80c">